assets/web-ui/js/widget.js

```diff
@@ -1620,7 +1660,21 @@ export class FlatProfile extends Widget {
 <table width="${this.container.width() - 20}px">
 <thead>
     <tr>
-        <th rowspan="3" class="sortable" data-sort="name">Function</th>
+        <th rowspan="3" class="sortable" data-sort="name">
```

Comment:
The search should really be placed at top IMO since it has a global impact (i.e. on all widgets). My first comment was that its width could and should be optimized, but I've never implied that it should be moved elsewhere.

Reply:
Ok. I changed the view according with your original suggestion.

![Selection_447](https://user-images.githubusercontent.com/3192770/179349443-afb9958b-d346-49fd-872f-6e3c5936aca4.png)
![Selection_448](https://user-images.githubusercontent.com/3192770/179349449-8f67a469-14a1-414a-8ea6-7eaa1df66a97.png)

Reply:
Thanks @ilnytskyi I'll try to test it ASAP